### ISSUE TITLE
Type not found

### ISSUE THREAD

**Issue by FuriouZz** [vshaxe/vshaxe#286]
Autocomplete does not seem to work...It works on scenes/MainScene.hx, but not on files in entities folder. Haxe output "Type not found".

Haxe version : 4.0.0.preview.4
Platform: Windows 10
Visual Studio Code: 1.25.1

![screenshot_1](https://user-images.githubusercontent.com/1800468/43986185-22ab03a8-9d0e-11e8-851c-1400b8560017.png)

**Comment by Gama11** [comment]
It's interesting how the class name is lowercased in the error message (`bullet` instead of `Bullet`). Do you perhaps have it misspelled that way somewhere? Does the project compile at all?

**Comment by FuriouZz** [comment]
The project compile without any error.

**Comment by Gama11** [comment]
Any way for me to reproduce this?

**Comment by FuriouZz** [comment]
I do not really know why it happens. I opened my project in VSCode on my Macbook, I do not have any problem. It only happens on Windows and all the time. Is the strange that the filepath is fully on lowercase, no?

**Comment by Gama11** [comment]
The path after "display position"? I think that's expected, Haxe normalizes file paths internally (which on Windows also means lowercasing them).

Could you share the project so I could try to reproduce the problem?

**Comment by FuriouZz** [comment]
[infinity-haxe.zip](https://github.com/vshaxe/vshaxe/files/2280157/infinity-haxe.zip)

**Comment by Gama11** [comment]
Thanks. Unfortunately, I can't reproduce the problem even with that (using Windows 10).

![](https://i.imgur.com/ZS3tMC8.png)

**Comment by FuriouZz** [comment]
I installed WSL (Windows Subsystem flr Linux) and created haxe/haxelib/hl aliases. This can cause any issue?

**Comment by Gama11** [comment]
Perhaps, I've never tried that.

**Comment by FuriouZz** [comment]
I did a complete installation of VSCode and Haxe 4.0.0.preview.4, but it still does not work =(. Completion works on MainScene.hx, but not Bullet.hx and Player.hx...

**Comment by FuriouZz** [comment]
OK, I renamed `entities` folder to `$entities`, created a new one call `entities` and move every files on it. And it works. Maybe it is the same issue #117 ?

**Comment by Gama11** [comment]
Not sure, it doesn't sound like there were UTF-8 characters involved in your case? Anyway, glad you got it working.

**Comment by kLabz** [comment]
Same setup (WSL + Haxe 4.0.0 preview 4), same issue.

**Comment by FuriouZz** [comment]
OK, I understood where the problem came from. My project was created on OSX at the begining. When I started to work on Windows, I cloned the project with git from WSL (Windows-SubSystem Linux). Linux seems to be case-sensitive and Windows not. I use git from WSL and Haxe from Windows.

To fix the problem, I cloned my project with git windows.

If it can help you @kLabz.

**Comment by kLabz** [comment]
Yes, I used this workaround (copied the files from windows to make windows own the files), but the problem comes back after a while =/

I tracked the issue inside vshaxe and came to the conclusion that vshaxe didn't seem to do anything wrong, but when it calls haxe, haxe itself fails to find the file. Something may be possible within haxe, but I do not have the environment to test further.

I also tried to make vshaxe use the haxe from WSL (which can successfully run the commands haxe.exe fails to run), but to no avail.
